### ISSUE TITLE
Cannot run version 0.121.0

### ISSUE THREAD

**Issue by martinholmer** [PolicyEngine/openfisca-us#1216]
When I install the 0.121.0 version, I get the following error having to do with the new `short_term_capital_losses` variable:
```
(openfisca-us) OpenFisca-US% ./valtest.sh p21 US TAOUT 
START p21 validation test for US at Fri Aug  5 10:24:46 EDT 2022
ERROR:openfisca_core.taxbenefitsystems.tax_benefit_system:Unable to load OpenFisca variables from file "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_us/variables/household/income/person/capital_gains/short_term_capital_losses.py"
Traceback (most recent call last):
  File "../conduct_test.py", line 482, in <module>
    sys.exit(main())
  File "../conduct_test.py", line 422, in main
    sim = Microsimulation(reform=(), dataset=vdset, year=vdset_year)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_us/api/microsimulation.py", line 19, in __init__
    super().__init__(reform, dataset, year)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_tools/microsimulation.py", line 68, in __init__
    self.load_dataset(self.dataset, self.year)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_tools/microsimulation.py", line 97, in load_dataset
    self.system = self.tax_benefit_system()
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_us/system.py", line 34, in __init__
    os.path.join(COUNTRY_DIR, "variables")
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_core/taxbenefitsystems/tax_benefit_system.py", line 219, in add_variables_from_directory
    self.add_variables_from_directory(subdirectory)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_core/taxbenefitsystems/tax_benefit_system.py", line 219, in add_variables_from_directory
    self.add_variables_from_directory(subdirectory)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_core/taxbenefitsystems/tax_benefit_system.py", line 219, in add_variables_from_directory
    self.add_variables_from_directory(subdirectory)
  [Previous line repeated 1 more time]
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_core/taxbenefitsystems/tax_benefit_system.py", line 216, in add_variables_from_directory
    self.add_variables_from_file(py_file)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_core/taxbenefitsystems/tax_benefit_system.py", line 205, in add_variables_from_file
    self.add_variable(pot_variable)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_core/taxbenefitsystems/tax_benefit_system.py", line 150, in add_variable
    return self.load_variable(variable, update = False)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_core/taxbenefitsystems/tax_benefit_system.py", line 137, in load_variable
    variable = variable_class(baseline_variable = baseline_variable)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_tools/model_api/model_api.py", line 39, in __init__
    CoreVariable.__init__(self, baseline_variable=baseline_variable)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_core/variables/variable.py", line 121, in __init__
    self.reference = self.set(attr, 'reference', setter = self.set_reference)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_core/variables/variable.py", line 157, in set
    value = setter(value)
  File "/Users/mrh/anaconda3/envs/openfisca-us/lib/python3.7/site-packages/openfisca_core/variables/variable.py", line 193, in set_reference
    raise TypeError('The reference of the variable {} is a {} instead of a String or a List of Strings.'.format(self.name, type(reference)))
TypeError: The reference of the variable short_term_capital_losses is a <class 'dict'> instead of a String or a List of Strings.
Files p21-US.res-actual and p21-US.res-expect differ
FAILED p21-US validation test
END   p21 validation test for US at Fri Aug  5 10:25:58 EDT 2022
```

Is there something not quite right about the changes in PR #1214, or do I need to somehow change my code in order to use version 0.121.0?

**Comment by nikhilwoodruff** [comment]
Ah, sorry Martin, I know exactly what this is. I updated OpenFisca-Tools to cope with references that include a title and a URL, but I didn't update the dependency in OpenFisca-US. Fixed by #1217 .

**Comment by martinholmer** [comment]
@nikhilwoodruff,  Thanks for the quick response.
